### PR TITLE
Docs: instance.publish(packet) expects an aedes-packet object

### PR DESCRIPTION
**Pull request by eladnava**
Updated the docs to match the new parameter expectation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -143,7 +143,7 @@ It supports backpressure.
 ### instance.publish(packet, done)
 
 Publish the given packet to subscribed clients and functions. A packet
-must be valid for [mqtt-packet](http://npm.im/mqtt-packet).
+must be valid for [aedes-packet](http://npm.im/aedes-packet).
```
Comment (mcollina) [inline README.md]: In aedes-packet, everything is defaulted:

https://github.com/mcollina/aedes-packet/blob/master/packet.js#L4-L11.

I think a much better definition can be: a packet must have the following properties...

Comment (eladnava) [inline README.md]: @mcollina I think you have outlined it well here for the Publish packet:
https://www.npmjs.com/package/mqtt-packet#publish

Should I just copy over that explanation?

Comment (mcollina) [inline README.md]: I think so, yes!

Comment (eladnava) [inline README.md]: @mcollina Updated the PR =) Let me know if anything is missing.

**Comment by mcollina** [comment]
Perfect, thanks!

**Pull request merged**